packages/react/src/components/Chat/ChatItem.tsx

```diff
@@ -89,23 +90,62 @@ class ChatItem extends UIComponent<ReactProps<ChatItemProps>, any> {
   }
 
   private renderChatItem(styles: ComponentSlotStylesPrepared) {
-    const { message, gutter, contentPosition } = this.props
+    const { gutter, contentPosition } = this.props
     const gutterElement =
       gutter &&
       Box.create(gutter, {
         defaultProps: { className: ChatItem.slotClassNames.gutter, styles: styles.gutter },
       })
 
+    const messageElement = this.getMessageAugmentedWithChatMessageProps(styles)
+
     return (
       <>
         {contentPosition === 'start' && gutterElement}
-        {Box.create(message, {
-          defaultProps: { className: ChatItem.slotClassNames.message, styles: styles.message },
-        })}
+        {messageElement}
         {contentPosition === 'end' && gutterElement}
       </>
     )
   }
+
+  getMessageAugmentedWithChatMessageProps = styles => {
+    const { message, attached } = this.props
+    const messageElement = Box.create(message, {
+      defaultProps: {
+        className: ChatItem.slotClassNames.message,
+        styles: styles.message,
+      },
+    })
+
+    // the box element is ChatMessage
+    if (this.checkIfElementIsChatMessage(messageElement)) {
+      return this.cloneElementWithCustomProps(messageElement, { attached })
+    }
+
+    // the children element is ChatMessage
```

Comment:
given that `children`, theoretically, may be a function, the following comment would be more correct:

> the children is ChatMessage element

the same applies to the `content`